contrib/cluster-backup/restore.sh

```diff
@@ -141,7 +141,7 @@ main() {
     for ns in ${NAMESPACES}; do
         if [ -e "${TEMP_DIR}/${ns}" ] && [ -d "${TEMP_DIR}/${ns}" ]; then
         echo "Restoring resources for '${ns}' Namespace..."
-        kubectl create -f <(echo $(cat ${TEMP_DIR}/namespaces.json | jq -r --arg NS "$ns" 'select( .items[].metadata.name | contains($NS))'))
+        kubectl create -f <(echo $(cat ${TEMP_DIR}/namespaces.json | jq -r --arg NS "$ns" '.items[] | select(.metadata.name | contains($NS))'))
```

Comment:
@skloss Wow! So, it was (mistakenly) restoring all the namespaces at once regardless of what was said to be restored?

Reply:
@mumoshu  Yes,exactly and also it wasn't creating any ressource inside the namespace

Reply:
tested it also on two cluster's